File: app/src/main/java/com/chooloo/www/callmanager/ui/about/AboutMvpView.java
package com.chooloo.www.callmanager.ui.about;

import com.chooloo.www.callmanager.ui.base.MvpView;

public interface AboutMvpView extends MvpView {
    void onSetup();

    void openChangelog();

    void openSource();

    void sendEmail();

    void reportBug();

    void rateApp();

    void donate();
}

File: app/src/main/java/com/chooloo/www/callmanager/ui/about/AboutMvpView.kt
package com.chooloo.www.callmanager.ui.about

import com.chooloo.www.callmanager.ui.base.MvpView

interface AboutMvpView : MvpView {

    fun openSource()
    fun sendEmail()
    fun reportBug()
    fun rateApp()
    fun donate()
}